### PR TITLE
Track velocity norms as dynamic proxies in Axial stretching and Timoshenko examples

### PR DESCRIPTION
**Pull request by tp5uiuc**
Adds functionality to track velocity of the rod to check if the simulation is close to reaching the quasi-steady state. 

Apologies for adding a new branch to the main repo, I was in a hurry.

### CODE REVIEW

File: examples/TimoshenkoBeamCase/timoshenko.py [hunk 1]
```diff
@@ -85,9 +85,34 @@ class TimoshenkoBeamSimulator(BaseSystemCollection, Constraints, Forcing):
...
     timoshenko_sim.add_forcing_to(unshearable_rod).using(
-        EndpointForces, 0.0 * end_force, end_force, ramp_up_time=final_time / 2.0
+        EndpointForces, 0.0 * end_force, end_force, ramp_up_time=100.0# final_time / 2.0
     )
 
+# Add call backs
+class VelocityCallBack(CallBackBaseClass):
+    """
+    Call back function for continuum snake
+    """
```
Comment (bhosale2) [inline examples/TimoshenkoBeamCase/timoshenko.py:95]: Can you correct this docstring?

**Review by bhosale2 — CHANGES_REQUESTED** [review]
One minor change.

**Review by armantekinalp — APPROVED** [review]
LGTM

File: examples/TimoshenkoBeamCase/timoshenko.py (hunk 1 above)
Comment (tp5uiuc) [reply examples/TimoshenkoBeamCase/timoshenko.py]: My bad. Updated in [dfa9869](https://github.com/GazzolaLab/PyElastica/pull/97/commits/dfa986958533d4919c4436f30dde3327ab629a00).

**Review by bhosale2 — APPROVED** [review]
LGTM.

**Pull request merged**